packages/react-native-storybook/package.json

```diff
@@ -20,6 +20,12 @@
   },
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "exports": {
```

Comment:
sprawdzic czy potrzebne

Reply:
Both fields serve different purposes and are often used together for compatibility reasons:
main and types are traditional fields used by older Node.js versions and TypeScript
exports is a newer Node.js feature that provides more precise control over package entry points
For maximum compatibility, keep both. The main field ensures older tools can use your package, while exports gives you better control with newer environments.
If you only target newer Node.js environments (12+), you could technically use just exports, but including both is the safer approach for wider compatibility.